From ab8e4a722d0742ef97ecd2fc79dfd251e94a8fb7 Mon Sep 17 00:00:00 2001
From: Francisco Munoz <b37752@freescale.com>
Date: Thu, 12 Jan 2012 19:13:36 -0600
Subject: [PATCH 0896/2508] ENGR00171946 SPI nor chip select update

SPI nor cannot be probed, due to a previous patch which
modified its chip select settings.

Signed-off-by: Francisco Munoz <b37752@freescale.com>
---
 arch/arm/mach-mx6/board-mx6q_sabreauto.c |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/arch/arm/mach-mx6/board-mx6q_sabreauto.c b/arch/arm/mach-mx6/board-mx6q_sabreauto.c
index 28f37cc..1000e12 100644
--- a/arch/arm/mach-mx6/board-mx6q_sabreauto.c
+++ b/arch/arm/mach-mx6/board-mx6q_sabreauto.c
@@ -595,7 +595,7 @@ static struct spi_board_info m25p32_spi0_board_info[] __initdata = {
 		.modalias = "m25p80",
 		.max_speed_hz = 20000000,
 		.bus_num = 0,
-		.chip_select = 1,
+		.chip_select = 0,
 		.platform_data = &m25p32_spi_flash_data,
 	},
 #endif
-- 
1.7.7.4

